mock/mock.go

```diff
@@ -28,6 +28,11 @@ func (e *MockEnv) Getenv(key string) string {
 	return e.data[key]
 }
 
+func (e *MockEnv) LookupEnv(key string) (string, bool) {
+	val, ok := e.data[key]
+	return val, ok
```

Comment:
Also, you can skip the variables here and return `e.data[key]` directly.

Reply:
hmm I tried it but I get a compilation error: `not enough arguments to return`. I think go does not recognize that I want to return the `ok` variable as well from map access, so I did this explicitly